framework/include/problems/FEProblemBase.h

```diff
@@ -700,9 +700,9 @@ class FEProblemBase : public SubProblem, public Restartable
 
   virtual void
   addFVBC(const std::string & fv_bc_name, const std::string & name, InputParameters & parameters);
-  void addFVInterfaceKernel(const std::string & fv_ik_name,
-                            const std::string & name,
-                            InputParameters & parameters);
+  virtual void addFVInterfaceKernel(const std::string & fv_ik_name,
+                                    const std::string & name,
+                                    InputParameters & parameters);
```

Comment:
why

Reply:
All of those methods have to be virtual so that the DumpObjectsProblem can override them to intercept the parameters of the added objects.